firefox-ios/Client/Frontend/Autofill/Address/AddressAutofillToggle.swift

```diff
@@ -94,6 +68,36 @@ struct AddressAutofillToggle: View {
         }
     }
 
+    private func mainContent() -> some View {
```

Comment:
nit: I suggest computed properties instead of functions to keep the `var body: some View` pattern. `main` should suffice.

Reply:
Thank you, @razvanlitianu. I'll update as soon as possible.